### PR TITLE
Allow `interface_body` and `enum_body` to consist of just a `;`.

### PR DESCRIPTION
**Pull request by AlekseyTs**
https://github.com/dotnet/csharplang/blob/main/proposals/primary-constructors.md#syntax

### CODE REVIEW

**Comment by AlekseyTs** [comment]
@cston, @jjonescz, @dotnet/razor-compiler Please review.

File: src/Compilers/CSharp/Portable/Syntax/Syntax.xml [hunk 1]
```diff
@@ -3417,7 +3417,7 @@
       <Kind Name="SemicolonToken"/>
     </Field>
   </Node>
-  <Node Name="InterfaceDeclarationSyntax" Base="TypeDeclarationSyntax">
+  <Node Name="InterfaceDeclarationSyntax" Base="TypeDeclarationSyntax" SkipConvenienceFactories="true">
```
Comment (jcouv) [inline src/Compilers/CSharp/Portable/Syntax/Syntax.xml:3420]: >SkipConvenienceFactories="true" [](http://example.com/codeflow?start=71&length=31)

Should we add this to RecordDeclarationSyntax too?

Comment (AlekseyTs) [reply src/Compilers/CSharp/Portable/Syntax/Syntax.xml:3420]: > Should we add this to RecordDeclarationSyntax too?

We are not making any changes for records, so no changes necessary, even if they wouldn't make any difference (which is unlikely).

**Review by jcouv — APPROVED** [review]
LGTM Thanks (iteration 1)

**Comment by AlekseyTs** [comment]
@cston, @jjonescz, @dotnet/razor-compiler For the second review

**Review by jjonescz — APPROVED** [review]

**Pull request merged**